app/BUILD.bazel

```diff
@@ -123,6 +123,9 @@ VIEW_MODELS_WITH_RESOURCE_IMPORTS = [
     "src/main/java/org/oppia/android/app/help/HelpListViewModel.kt",
     "src/main/java/org/oppia/android/app/help/faq/FAQListViewModel.kt",
     "src/main/java/org/oppia/android/app/home/HomeViewModel.kt",
+    "src/main/java/org/oppia/android/app/home/topiclist/TopicSummaryViewModel.kt",
+    "src/main/java/org/oppia/android/app/home/promotedlist/ComingSoonTopicListViewModel.kt",
```

Comment:
This doesn't look like it has any resource imports, should remove this

Reply:
Added this import in VIEW_MODELS.